tools/aws_lab_setup/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -1,11 +1,18 @@
 ec2_name_prefix: TRAINING
-ec2_security_group: insecure_all
+ec2_security_group: ""
```

Comment:
this makes total sense, I think the problem was that I kept trying to use  `is not defined` (which does not exist) and the proper terminology is `is undefined`.  Rather than troubleshooting it I just defaulted the values to keep moving on.  This was a trivial fix with commit https://github.com/network-automation/lightbulb/commit/a478e27d623d52bbbe72523f8665675da96560c3